### PR TITLE
RUN-3027 Fixed issue where RVM bus originated errors were not raised to the adapters.

### PR DESCRIPTION
**Pull request by rdepena**
[Test results](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/592f315c998bc04e20cc6c95)

### CODE REVIEW

File: src/browser/api/system.js [hunk 1]
```diff
@@ -566,12 +566,15 @@ module.exports.System = {
...
 
-        const publishSuccess = rvmBus.publish(rvmMessage);
-
-        if (publishSuccess) {
-            cb(null, downloadId);
+        const publishSuccess = rvmBus.publish(rvmMessage, response => {
+            if (response.error) {
+                cb(new Error(response.error));
+            } else {
+                cb();
```
Comment (HarsimranSingh) [inline src/browser/api/system.js]: Is this supposed to be `cb(null, downloadId)`?

Comment (rdepena) [inline src/browser/api/system.js]: That was the old style, as of [This PR](https://github.com/openfin/javascript-adapter/pull/303) the download ID's are generated by the renderer so no need to provide them.

Comment (HarsimranSingh) [inline src/browser/api/system.js]: Ah gotcha

**Comment by rdepena** [comment]
[updated test](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5931d95a998bc04e20cc6caf)

**Comment by HarsimranSingh** [comment]
👍

**Pull request merged**